### DISCUSSION TITLE
Feature: Move all windows for an application in one command

### DISCUSSION THREAD

**Discussion by mfbx9da4** [rxhanson/Rectangle#923] [General]
Very often I want to move all notion windows or all Xcode windows to the left or to the right. 

If all windows are not moved, subsequent usage of CMD+tab leads to the entire screen being eclipsed by the application.

**Comment by rxhanson** [comment]
Moving all windows to the left/right halves or next/prev displays are commands that are available in [Rectangle Pro](https://rectangleapp.com/pro). I don't have plans to add these into Rectangle, but this brings up the point of where my stance is between the two.

I see Rectangle as community driven wherever anyone wants to contribute, with the limitation that the UI must remain simple. Therefore, I don't want to add more window commands to the UI, and any features that could get added to the UI must either really make sense or have a lot of demand. So, if someone created a pull request with the functionality that you desire, I would merge it in but it would have to be configured via Terminal commands. Just want to be clear that I don't try to keep Rectangle slim for the benefit of Rectangle Pro, I'm just trying to make both of them the best products they can be: one that strives for simplicity, centered around the community and one that has any/every window management idea that I like and want to implement in it.
